internal/dag/builder.go

```diff
@@ -1094,6 +1104,23 @@ func (b *Builder) lookupUpstreamValidation(uv *projcontour.UpstreamValidation, n
 	}, nil
 }
 
+func (b *Builder) lookupDownstreamValidation(vc *projcontour.DownstreamValidation, namespace string) (*PeerValidationContext, error) {
+	if vc == nil {
```

Comment:
This never gets called.

Reply:
I removed the dead if branch `if vc == nil...`